src/graphnet/models/task/reconstruction.py

```diff
@@ -92,6 +92,18 @@ def _forward(self, x: Tensor) -> Tensor:
         return torch.pow(10, x[:, 0] + 1.0).unsqueeze(1)
 
 
+class EnergyReconstructionWithoutPowerTransform(Task):
+    """Reconstructs energy without power-10 transformation."""
+
+    # Requires one feature: untransformed energy
+    nb_inputs = 1
+
+    def _forward(self, x):
```

Comment:
`mypy` is giving an error because the method is lacking type annotations. See suggestion below.
```suggestion
    def _forward(self, x: Tensor) -> Tensor:
```

Reply:
seems sensible I'll try and make the commit.